apps/backend/src/modules/contributors/contributor.controller.ts

```diff
@@ -108,6 +110,21 @@ class ContributorController extends BaseController {
 				body: contributorPatchValidationSchema,
 			},
 		});
+
+		this.addRoute({
+			handler: (options) =>
+				this.split(
+					options as APIHandlerOptions<{
+						body: ContributorSplitRequestDto;
+						params: { id: string };
+					}>,
+				),
+			method: "PATCH",
+			path: ContributorsApiPath.SPLIT_$ID,
```

Comment:
Let's add `preHandler` to allow this action only if user has `Manage All Projects` permission